src/services/getPersonDates.ts

```diff
@@ -0,0 +1,34 @@
+import { useContext } from 'react';
+import { PeopleContext } from '../store/PeopleContent';
+
+function ExtractDates(name: string | null) {
+  const { people } = useContext(PeopleContext);
+  const parentOfKid = people.find(personEl => personEl.name === name);
+
+  if (!parentOfKid) {
+    return undefined;
+  }
+
+  const parentBirthDate = parentOfKid?.born;
+
+  const parentNameDivided = parentOfKid?.name
+    .split(' ')
+    .join('-')
+    .toLowerCase();
+
+  const parentDates = parentNameDivided + '-' + parentBirthDate;
+
+  return parentDates;
+}
```

Comment:
The `ExtractDates` function uses the `useContext` hook, which is not recommended outside of a React component or a custom hook. Consider refactoring this function to be a custom hook or passing the `people` data as an argument.